userspace/falco/webserver.h

```diff
@@ -25,6 +25,8 @@ limitations under the License.
 #include <memory>
 #include <thread>
 
+class falco_metrics;
```

Comment:
Is there a reason for the forward declaration?

Reply:
Mostly out of habits from other projects I work on.